src/main/java/frc/robot/commands/ControlPanelColorCommand.java

```diff
@@ -1,41 +1,39 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.common.ControlPanelColorSensor.ControlPanelColor;
 import frc.robot.subsystems.ControlPanelSubsystem;
 
 public class ControlPanelColorCommand extends CommandBase {
-    private ControlPanelSubsystem controlPanel;
-    private ControlPanelColor turnToColor;
+  private ControlPanelSubsystem controlPanel;
+  private float direction;
 
-    public ControlPanelColorCommand(ControlPanelColor turnToColor) {
+    public ControlPanelColorCommand() {
         // TODO: Why won't this line do what we want? Let's think about it
         controlPanel = new ControlPanelSubsystem();
-        this.turnToColor = turnToColor;
     }
 
-    @Override
-    public void initialize() {
-        // Extend the solenoid if it isn't already
-        controlPanel.setSolenoid(true);
-    }
+  @Override
+  public void initialize() {
+    controlPanel.setSolenoid(true);
+    direction = Math.signum(controlPanel.getTurnToColor()
+        .ordinal() - controlPanel.getDetectedColor()
+        .ordinal());
```

Comment:
Initialize direction once. No need to calculate it on every loop in `execute`

Reply:
I think we had it do this in python in case it overshot its destination color (it could then move the other direction). Although it probably wouldn't overshoot in actuality.